packages/dart/noports_core/lib/src/sshnpd/sshnpd_impl.dart

```diff
@@ -141,6 +144,17 @@ class SshnpdImpl with AtClientBindings implements Sshnpd {
         DaemonFeature.supportsPortChoice.name: true,
         DaemonFeature.adjustableTimeout.name: true,
       },
+      'e2eeModes': {
```

Comment:
this doesn't actually belong in this PR but @XavierChanth I'd be interested in your input on naming and structure here